website/src/config.ts

```diff
@@ -11,7 +12,7 @@ let _config: WebsiteConfig | null = null;
 let _runtimeConfig: RuntimeConfig | null = null;
 
 function getConfigDir(): string {
-    const configDir = import.meta.env.CONFIG_DIR;
+    const configDir = process.env.CONFIG_DIR;
     if (typeof configDir !== 'string' || configDir === '') {
         throw new Error(`CONFIG_DIR environment variable was not set during build time, is '${configDir}'`);
```

Comment:
If we use `process` now, the error message is not really correct anymore - the env variable will not be baked into the code at build time anymore but instead actually be read at runtime.

Reply:
Yeah I was just trying out stuff